demos/common/cpp/utils/include/utils/ocv_common.hpp

```diff
@@ -20,43 +20,44 @@
 * @param batchIndex - batch index of an image inside of the blob.
 */
 template <typename T>
-void matU8ToBlob(const cv::Mat& orig_image, const InferenceEngine::Blob::Ptr& blob, int batchIndex = 0) {
+void matToBlob(const cv::Mat& orig_image, const InferenceEngine::Blob::Ptr& blob, int batchIndex = 0) {
```

Comment:
I still insist on making this function non-templated. It should evaluate type T from the type of blob (getTensorDesc().getPrecision()) We should check if it will affect performance though, but I think it should not slow down.
You could make helper function templates, but main function (matToBlob) should not take template arguments as it might lead to bugs later.

Reply:
Updated, the `matToBlob` is no longer a template function. Is the current implementation okay?
I checked the performance metrics, the preprocessing time did not change with this patch, so I think it does not affect the performance.